test/ui/cypress/e2e/deployment_driver.cy.js

```diff
@@ -18,38 +11,101 @@ describe('Deployment driver web UI', () => {
     cy.get('@main').contains('Deployment Steps')
     cy.get('@main').get('div.deployProgress').as('statusArea')
     cy.get('@statusArea').get('button.cancelButton').contains('Cancel Deployment')
+
+    cy.get('.pf-c-brand').should('be.visible')
+    cy.get('.pf-c-alert__title')
+      .contains('You currently have a subscription to Ansible Automation Platform')
+    cy.get('p').contains('To manage or setup new subscription, visit the')
+    cy.get('p > .pf-c-button')
+      .contains('Red Hat Hybrid Cloud Console')
+      .should('have.attr', 'href', 'https://console.redhat.com/')
+
+    // Check the H1 title
+    cy.get('h1')
```

Comment:
This is too generic, it will look for any `<h1>` tag, it needs to be more specific, see test above.

Reply:
@tznamena , do you have any good way to replace it?

Reply:
Leave it as it is. Will have a new PR to refine it after @tznamena suggest a better element selector for it.